engine/src/main/java/org/terasology/config/RenderingConfig.java

```diff
@@ -328,7 +328,7 @@ public void setVignette(boolean vignette) {
     }
 
     public boolean isMotionBlur() {
-        return motionBlur && !isOculusVrSupport();
+        return motionBlur && !isVrSupport();
```

Comment:
I would say that motion blur is incompatible with VR, as are several other effects. However, I think the best way to handle such cases is to take the extra conditional out and add a VR preset in a future PR.

I'll go ahead and take this out.